lib/loggly/common.js

```diff
@@ -209,14 +227,18 @@ common.loggly = function () {
           }
         })
         .catch(function(err) {
-           return handleRequestError(input, err);
+          return handleRequestError(input, err);
         });
     } catch (ex) {
       onError(ex);
     }
   }
 
   function handleRequestError(input, err) {
+    if(err.response && groupedHttpCodes.indexOf(err.response.status) != -1) {
+      console.log(String(err))
+      return true;
```

Comment:
does it mean that in case of error we will return empty response?

Reply:
Just thought to only show the error on console. Still let me check what can be improved here.

Reply:
I have handled the exception post logging in winston-loggly-bulk. You can **close this PR (Don't merge)**. Please review this PR https://github.com/loggly/winston-loggly-bulk/pull/98